### PR TITLE
Cpplint namespace

### PR DESCRIPTION
**Pull request by phlptp**
### Summary

If merged this pull request will resolve cpplint build/namespace issues.

### CODE REVIEW

**Comment by phlptp** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- benchmarks/helics/echoMessageBenchmarks.cpp  1
- benchmarks/helics/pholdBenchmarks.cpp  1
- benchmarks/helics/filterBenchmarks.cpp  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GMLC-TDC/HELICS/pullRequest?prid=5366126&bid=17583850)

**Comment by phlptp** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- benchmarks/helics/pholdBenchmarks.cpp  1
- benchmarks/helics/filterBenchmarks.cpp  1
- benchmarks/helics/echoMessageBenchmarks.cpp  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GMLC-TDC/HELICS/pullRequest?prid=5366126&bid=17583850)

File: benchmarks/helics/RingTransmitMessageFederate.hpp [hunk 1]
```diff
@@ -12,7 +12,6 @@ SPDX-License-Identifier: BSD-3-Clause
 #include "helics/core/ActionMessage.hpp"
 
 using helics::operator"" _t;
```
Comment (nightlark) [inline benchmarks/helics/RingTransmitMessageFederate.hpp:14]: Is this still needed here?

**Comment by phlptp** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- benchmarks/helics/pholdBenchmarks.cpp  1
- benchmarks/helics/echoMessageBenchmarks.cpp  1
- benchmarks/helics/filterBenchmarks.cpp  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GMLC-TDC/HELICS/pullRequest?prid=5366126&bid=17583850)

**Comment by phlptp** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- benchmarks/helics/filterBenchmarks.cpp  1
- benchmarks/helics/echoMessageBenchmarks.cpp  1
- benchmarks/helics/pholdBenchmarks.cpp  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GMLC-TDC/HELICS/pullRequest?prid=5366126&bid=17583850)

**Pull request merged**